src/providers/haskell.rs

```diff
@@ -27,6 +27,16 @@ impl Provider for HaskellStackProvider {
         let mut setup_phase = SetupPhase::new(vec![Pkg::new("stack")]);
         setup_phase.add_apt_pkgs(vec![
             "libgmp-dev".to_string(),
+            "libnuma1".to_string(),
+            "libnuma-dev".to_string(),
+            "libtinfo-dev".to_string(),
+            "libtinfo5".to_string(),
+            "libc6-dev".to_string(),
+            "libtinfo6".to_string(),
+            "llvm-11".to_string(),
+            "clang".to_string(),
+            "ninja-build".to_string(),
+            "zlib1g-dev".to_string(),
```

Comment:
can these be installed only if arm is detected? Since it currently works for non-arm

Reply:
sounds good I will update it accordingly.